### DISCUSSION TITLE
Where can i find playmaker and add it to references

### DISCUSSION THREAD

**Discussion by Tduck5819** [piotrulos/MSCModLoader#188] [Help me] [answered]
I want to add the reference playmaker but where is the file located at, do if have to download it?

**Comment by piotrulos** [comment, marked as answer]
In game files

**Reply by Tduck5819** [reply]
Ok thx one question i want to find the gameobject of the glass at the backyard from the drunk house new. Is there a mod or something to find specific gameobjects.
